### DISCUSSION TITLE
Standardize localization with GNU gettext format

### DISCUSSION THREAD

**Discussion by nightm4re94** [gurkenlabs/litiengine#453] [Ideas]
In order to make translators' lives easier, we could apply the [GNU gettext portable object format](https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html#PO-Files) to our game localization files. Then, they could be edited more conveniently with tools such as [poedit](https://poedit.net/). I think the PO format is superior to XLIFF in terms of readability by humans, which is why I wouldn't consider using XLIFF.

We could also have a look at how the [mini2Dx](https://github.com/mini2Dx/gettext) java game library handles gettext PO files. Conveniently, the library provides a Gradle plugin to automatically parse source files and generate templates. 

Read more about the PO format and its specifications:
http://pology.nedohodnik.net/doc/user/en_US/ch-poformat.html
https://www.drupal.org/node/1814954

**Comment by steffen-wilke** [comment]
https://docs.weblate.org/en/weblate-3.10/formats.html#gettext
